airflow-core/src/airflow/api_fastapi/core_api/datamodels/dag_run.py

```diff
@@ -57,7 +57,7 @@ class DAGRunClearBody(StrictBaseModel):
     only_failed: bool = False
     run_on_latest_version: bool = Field(
         default=False,
-        description="(Experimental) Run on the latest bundle version of the DAG after clearing the DAG Run.",
+        description="(Experimental) Run on the latest bundle version of the Dag after clearing the Dag Run.",
```

Comment:
Curious: why these changes?

Reply:
We no longer use DAG in Airflow to mean the Directed Acyclic Graph as it could be cyclic for AI jobs in the future: https://lists.apache.org/thread/lktrzqkzrpvc1cyctxz7zxfmc0fwtq2j, https://lists.apache.org/thread/8k338stlkkp07ko3no70p2nng757kd1w,

Reply:
I thought it was the other way around, sounds good.